packages/ilios-common/addon/models/vocabulary.js

```diff
@@ -15,8 +15,7 @@ export default class Vocabulary extends Model {
   terms;
 
   async getTopLevelTerms() {
-    const terms = await this.terms;
-    return filter(terms.slice(), async (term) => {
+    return filter(this.terms, async (term) => {
```

Comment:
I'm not sure about this one. Does `RSVP.filter` resolve the first argument `this.terms` when it is a promise? Documentation at https://github.com/tildeio/rsvp.js/ doesn't even cover `filter` so I'm not sure.

Reply:
Nevermind, I found where it [looks like that happens](https://github.com/tildeio/rsvp.js/blob/master/lib/rsvp/filter.js#L138)